webdriver-spec.html

```diff
@@ -975,26 +975,14 @@
 
  <tr>
   <td>GET</td>
-  <td>/session/{<var>session id</var>}/window/size</td>
-  <td><a>Get Window Size</a></td>
+  <td>/session/{<var>session id</var>}/window/dimensions</td>
+  <td><a>Get Window Dimensions</a></td>
```

Comment:
`Get Window Rect` would be in keeping with `Get Element Rect`. "Dimensions" implies only width and height.